src/Resources/views/standard_layout.html.twig

```diff
@@ -198,6 +198,7 @@ file that was distributed with this source code.
                         {% block sonata_side_nav %}
                             {% block sonata_sidebar_search %}
                                 {% if sonata_config.getOption('search') %}
+                                    {# NEXT_MAJOR: Replace form with symfony form #}
```

Comment:
What's blocking to do it now ?

Reply:
It requires a little bit more refactoring, because this template is loaded for every admin page.

We should either inject the new form using something like this https://github.com/sonata-project/SonataAdminBundle/pull/7482/files or we could render a sonata block.